package.json

```diff
@@ -13,9 +13,12 @@
     "lint:fix": "npx eslint . --fix"
   },
   "dependencies": {
-    "@mui/material": "^5.8.3",
+    "@emotion/react": "^11.10.4",
+    "@emotion/styled": "^11.10.4",
+    "@mui/icons-material": "^5.10.9",
+    "@mui/material": "^5.10.11",
     "@mui/styles": "^5.8.3",
-    "@prosopo/i18n": "0.0.1",
```

Comment:
This should be set to `0.0.2` as the next minor version will not be pulled for an unstable package.